tiledb/cf/__init__.py

```diff
@@ -51,6 +51,12 @@ def cli():
     show_default=True,
     help="Recursively convert all groups contained in the input group path.",
 )
+@click.option(
+    "--coords-to-dims/--coords-to-attrs",
+    default=True,
```

Comment:
Using `--coords-to-dims` in the CLI will make it true and `--coords-to-attrs` will make it false. If neither flag is provide, then it will default to true.